usr/bin/whonix-workstation-firewall

```diff
@@ -80,6 +83,28 @@ variables_defaults() {
     fi
   fi
 
+  if [ -z "${LOCAL_NET_IP6:-}" ]; then
+    if test -f /usr/share/qubes/marker-vm; then
+      LOCAL_NET_IP6="\
+            ::1/128 \
+            ::ffff:127.0.0.0/120 \
+            ::ffff:10.137.0.0/112 \
+            ::ffff:10.138.0.0/112 \
+         "
+    else
+      ## 10.0.2.2/24: VirtualBox DHCP
+      ## IP HARDCODED unfortunately. Use a /etc/whonix_firewall.d configuration folder drop-in if you want to change it.
+      LOCAL_NET_IP6="\
+            ::1/128 \
```

Comment:
I think you probably intended to list `fd19:c33d:88bc::0/96` here too, like you did in [whonix-gateway-firewall](https://github.com/Whonix/whonix-firewall/pull/10/commits/e7de2a00f00a7e99a7fc9c089edc738434a5ed9d#diff-96c8c0c6b9457724a30a65dd7e4c73c58ae0f212ca2885dac2e18a8752f479f2R179).

Reply:
Good catch, added.